src/grid.jl

```diff
@@ -279,4 +279,71 @@ function _is_true_neighbor(neighbor::NTuple{D,Int}, stencil::NTuple{D,Int}, grid
     return all(d -> 1 <= neighbor[d] <= get_ndims(grid,d), 1:D) && any(d -> stencil[d] != 0, 1:D)
 end
 
-finalize_grid(::DistributedGrid; finalize_MPI = true) = finalize_MPI && MPI.Finalize()
\ No newline at end of file
+finalize_grid(::DistributedGrid; finalize_MPI = true) = finalize_MPI && MPI.Finalize()
+
+function gather_array(array::AbstractArray{T,D}, grid::DistributedGrid{D}, root::Integer = 0) where {T,D}
```

Comment:
Can you add the following documentation:
```
"""
    gather_array(array::AbstractArray{T,D}, grid::DistributedGrid{D}, root::Integer = 0) where {T,D}

Establish parallel communication to gather a distributed array on the `root` processor.
"""
```

Reply:
Done